### DISCUSSION TITLE
Downloaded playlists

### DISCUSSION THREAD

**Discussion by blueberrie231** [fast4x/RiMusic#2797] [Q&A]
Is there an option to only see the playlists you have downloaded? I keep forgetting which ones I have downloaded and it would be helpful to have a "downloaded" button if there isn't one already

**Comment by GeorgSommer** [comment]
I guess you could create a playlist whereyou add the sownloaded.titles.
But rather i'd like to know whrre they are stored, so I could use them with other software.

**Reply by fast4x** [reply]
You can't because songs are internal crypted by player.

**Comment by fast4x** [comment]
> Is there an option to only see the playlists you have downloaded? I keep forgetting which ones I have downloaded and it would be helpful to have a "downloaded" button if there isn't one already

Adding that feature would slow the app down. You can access a playlist and look at the download icon if it is present in all songs

**Comment by aneesh1122** [comment]
> Is there an option to only see the playlists you have downloaded? I keep forgetting which ones I have downloaded and it would be helpful to have a "downloaded" button if there isn't one already

A workaround is to pin those playlists you've downloaded
